data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json

```diff
@@ -1,7 +1,7 @@
 {
-    "@odata.context" : "<%= basepath %>/$metadata#Systems/Members/$entity",
+    "@odata.context" : "<%= basepath %>/$metadata#ComputerSystemCollection.ComputerSystemCollection",
```

Comment:
This should be #ComputerSystem.ComputerSystem.
Remove 'Collection', that is in another file.